test/integration/controller.v2/trainjob_controller_test.go

```diff
@@ -237,7 +235,9 @@ var _ = ginkgo.Describe("TrainJob controller", ginkgo.Ordered, func() {
 				g.Expect(jobSet.Spec.Suspend).ShouldNot(gomega.BeNil())
 				g.Expect(*jobSet.Spec.Suspend).Should(gomega.BeTrue())
 				for _, rJob := range jobSet.Spec.ReplicatedJobs {
-					g.Expect(rJob.Template.Spec.Template.Spec.Containers[0].Image).Should(gomega.Equal(originImageName))
+					if rJob.Name == constants.JobTrainerNode {
+						g.Expect(rJob.Template.Spec.Template.Spec.Containers[0].Image).Should(gomega.Equal(originImageName))
+					}
 				}
 			}, util.Timeout, util.Interval).Should(gomega.Succeed())
 		})
```

Comment:
Could you add a case if we can build and deploy Torch JobSet like `ginkgo.It("Should succeed to create TrainJob with Torch TrainingRuntime", func() {`